### PR TITLE
New example for ESP-BOX: Touch Screen Color wheel 

### PR DESCRIPTION
**Pull request by espzav**
This is really simple example for beginners. I am working on more complex example too.

### CODE REVIEW

**Comment by espzav** [comment]
@tore-espressif and @VojtechBartoska PTAL

File: examples/touchscreen_colorwheel/main/main.c [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+ *
+ * SPDX-License-Identifier: CC0-1.0
+ */
+
+#include "esp_log.h"
+#include "bsp/esp-box.h"
+#include "lvgl.h"
+
+#define TAG "ESP-BOX"
+#define APP_DISP_DEFAULT_BRIGHTNESS 50
```
Comment (tore-espressif) [inline examples/touchscreen_colorwheel/main/main.c:12]: Why not 100?

FYI: Helper function `bsp_display_backlight_on(void);` can be used to set brightness to 100%

Comment (tore-espressif) [inline examples/touchscreen_colorwheel/main/main.c:12]: Why not 100?

FYI: Helper function `bsp_display_backlight_on(void);` can be used to set brightness to 100%

File: examples/touchscreen_colorwheel/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+| Supported Targets | ESP-BOX (ESP32-S3) |
+| ----------------- | ------------------ |
+
+# ESP-BOX Touch Screen Color Wheel Example
+
+This example demonstrates using ESP-BSP on ESP-BOX. There is a simple example, which changing background color by selecting on color wheel on touch screen.
```
Comment (tore-espressif) [inline examples/touchscreen_colorwheel/README.md:6]:
```suggestion
This example demonstrates usage of ESP-BOX Board Support Package. This is a single purpose example, which is focused on display + touch applications: user can change background color by selecting it on a color wheel on the touch screen. 
```
```suggestion
This example demonstrates using ESP-BSP on ESP-BOX. There is a simple example, which changing background color by selecting on color wheel on touch screen. 
```

Comment (tore-espressif) [inline examples/touchscreen_colorwheel/README.md:6]:
```suggestion
This example demonstrates usage of ESP-BOX Board Support Package. This is a single purpose example, which is focused on display + touch applications: user can change background color by selecting it on a color wheel on the touch screen. 
```
```suggestion
This example demonstrates using ESP-BSP on ESP-BOX. There is a simple example, which changing background color by selecting on color wheel on touch screen. 
```

**Review by tore-espressif — APPROVED** [review]
Few details, rest LGTM!

File: examples/touchscreen_colorwheel/main/main.c (hunk 1 above)
Comment (espzav) [reply examples/touchscreen_colorwheel/main/main.c:12]: I wanted use this function for show, how to use it.

File: examples/touchscreen_colorwheel/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+# https://docs.espressif.com/projects/esp-idf/en/latest/api-guides/build-system.html
+# The following five lines of boilerplate have to be in your project's
+# CMakeLists in this exact order for cmake to work correctly
+cmake_minimum_required(VERSION 3.5)
+
+include($ENV{IDF_PATH}/tools/cmake/project.cmake)
+project(touchscreen-colorwheel)
+
+# Following parameters can't be set via Kconfig
+idf_component_get_property(lvgl_lib lvgl__lvgl COMPONENT_LIB)
```
[the remark this answers is not in the document]
Comment (espzav) [reply examples/touchscreen_colorwheel/CMakeLists.txt:11]: yes, you are right

**Comment by tore-espressif** [comment]
@espzav feel free to squash & merge!

**Pull request merged**